documentation/sphinx/source/bulkload-user.rst

```diff
@@ -0,0 +1,155 @@
+###################
+BulkLoad User Guide
+###################
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :titlesonly:
+
+
+.. _bulkdump:
+
+Below we describe the :command:`BulkDump` and :command:`BulkLoad` 'fdbcli' commands and basic troubleshooting tips bulkloading.
+
+BulkDump
+==========
+
+Type :command:`help bulkdump` on the :command:`fdbcli` command-line to see the :command:`bulkdump` usage::
+
+    fdb> help bulkdump
+
+    bulkdump [mode|dump|status|cancel] [ARGs]
+
+    Bulkdump commands.
+
+    To set bulkdump mode: bulkdump mode [on|off]
+    To dump a range of key/values: bulkdump dump <BEGINKEY> <ENDKEY> <DIR>
+     where <BEGINKEY> to <ENDKEY> denotes the key/value range and <DIR> is
+     a local directory OR blobstore url to dump SST files to.
+    To get status: bulkdump status
+    To cancel current bulkdump job: bulkdump cancel <JOBID>
+
+
+To use the :command:`BulkDump` facility, you must first enable it as follows::
+
+    fdb> bulkdump mode on
+
+To dump all of the key/values in user space to a local directory::
+
+    fdb> bulkdump dump "" \xff /tmp/region
+    Received Job ID: 62d4548cf46dc0a9d06889ba3f6d1c08
+
+ To monitor the status of your running :command:`BulkDump` job, type::
+
+    fdb> bulkdump status
+    Running bulk dumping job: b20ce68b03a28d654ee948ca4b5d859a
+    Finished 1 tasks
+ 
+ The :command:`status` command will return variants on the above until finally after the job completes it will print::
+
+    fdb> bulkdump status
+    No bulk dumping job is running
+    
+To cancel a running job::
+
+    fdb> bulkdump cancel c9de5a364ecc2abb6f7a8bd890a175cf
+    Job c9de5a364ecc2abb6f7a8bd890a175cf has been cancelled. No new tasks will be spawned.
+
+ Only one :command:`BulkDump` (or :command:`BulkLoad`) job can be run at a time.
+
+.. _bulkload:
+
+BulkLoad
+==========
+:command:`BulkLoad` is the inverse of :command:`BulkLoad` but then adds a history dimension so you can see status or previous :command:`BulkLoad` runs.
```

Comment:
Seems that the sentence should be `BulkLoad` is the inverse of :command:`BulkDump`?

Reply:
Yes